### ISSUE TITLE
Reversed example

### ISSUE THREAD

**Issue by Zenexer** [php/doc-en#1278]
From manual page: https://php.net/function.include

---

Assuming file.php/file.txt is comparable it vars.php in Example 1, Example 3 has its working and non-working URLs reversed.  The first URL will work, while the second will not.

**Comment by cmb69** [comment]
It seems to me that the example would work as described and intended, but I agree that it is confusing, and especially

> Also, 'Works' here means that the variables $foo and $bar are available within the included file.

isn't quite up to date.

**Comment by Zenexer** [comment]
> It seems to me that the example would work as described and intended

Upon closer inspection, I think you're right and there's just a file missing from the example.  I suspect it was intended to contain something akin to the following:

``` php
<?php

printf('<?php extract(%s);', var_export($_GET, true));
```

This is an awful, insecure example and shouldn't be used in any documentation.

Here's a possible alternative, although this whole concept of remote inclusion with dynamic content scares the heck out of me:

``` php
<?php

echo "<?php\n\n";

echo "\$foo = 1234;\n";  //  $foo will be set to a hardcoded value; it won't be retrieved from the querystring

echo '$bar = ';  // Note that single quotes are used here.  '$bar' will be printed verbatim; it won't be expanded into a value
var_export($_GET['bar'] ?? null);  // $bar will be set to the value in the querystring
echo ';';
```

Ideally, the effective output from the remote server should be included:

``` php
<?php

$foo = 1234;
$bar = "2";
```

There are much better ways to accomplish this behavior that are less likely to result in security issues, but that's a separate matter.

**Comment by cmb69** [comment]
Yeah, I think you're right, but it's not unlikely that the example was written with `register_globals=On` in mind, or even was written before that was even an option.

> […] although this whole concept of remote inclusion with dynamic content scares the heck out of me:

Right. Fortunately, `allow_url_include` is deprecated as of PHP 7.4.0, but apparently is not removed as of PHP 8.0.0. I hope that'll happen for PHP 9.
